app/plugins/router.js

```diff
@@ -10,10 +10,10 @@ const routes = [].concat(
   require('../routes/submit-claim'),
   require('../routes/urn-result'),
   require('../routes/vet-name'),
-  require('../routes/vet-practice'),
   require('../routes/vet-rcvs'),
   require('../routes/verify-login'),
-  require('../routes/vet-visit-date')
+  require('../routes/vet-visit-date'),
+  require('../routes/check-answers')
```

Comment:
small one that I was always getting from Steve H :D 

In alphabetical order pls :)